{{cookiecutter.project_slug}}/backend/config/settings/local.py

```diff
@@ -11,7 +11,10 @@
     default="!!!SET DJANGO_SECRET_KEY!!!",
 )
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = ["localhost", "0.0.0.0", "127.0.0.1"]  # nosec
+ALLOWED_HOSTS = env(
+    "DJANGO_ALLOWED_HOSTS",
+    default=["backend"],
```

Comment:
Let's leavel the localhost options if someone wants to test locally outside the cluster.
```suggestion
    default=["backend", "localhost", "127.0.0.1"],
```